src/Kerbero.Common/Entities/NukiProviderAccount.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Kerbero.Common.Entities;
+
+public class NukiProviderAccount
+{
+  public string Token { get; set; }
```

Comment:
use a constructor or mark the field as nullable or use the not null assertion for the compiler